sandbox/apparmor/profile.go

```diff
@@ -251,3 +302,101 @@ var SnapConfineDistroProfilePath = func() string {
 
 	return ""
 }
+
+// SetupSnapConfineSnippets inspects the system and sets up local apparmor
+// policy for snap-confine.
+// Local policy is included by the system-wide policy. If the local policy
+// has changed then the apparmor profile for snap-confine is reloaded.
```

Comment:
I don't understand the last bit of the comment, as I don't see reloading code in this function. It seems reloading is a separate function?

Reply:
I'll remove that part of the comment, also there is currently no need for the function to be a `var` as it's not mocked currently.